src/graph.cc

```diff
@@ -554,24 +554,26 @@ bool ImplicitDepLoader::LoadDepFile(Edge* edge, const string& path,
     return false;
   }
 
-  // Preallocate space in edge->inputs_ to be filled in below.
-  vector<Node*>::iterator implicit_dep =
-      PreallocateSpace(edge, depfile.ins_.size());
-
-  // Add all its in-edges.
   for (vector<StringPiece>::iterator i = depfile.ins_.begin();
-       i != depfile.ins_.end(); ++i, ++implicit_dep) {
+       i != depfile.ins_.end(); ++i) {
     uint64_t slash_bits;
     if (!CanonicalizePath(const_cast<char*>(i->str_), &i->len_, &slash_bits,
                           err))
       return false;
-
```

Comment:
why did you remove this line?

Reply:
no strong reason or strong opinion, I'll just put it back in